Can't install node modules

Hi,

I've been stuck on the first install step (`npm install`) for a little while now, keep running into an issue I can't figure out how to fix, namely I get an error that says `TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".json" for .../bitsy-boilerplate/bitsy-paths.json`

Here's the full trace:
```
> bitsy-boilerplate@1.0.0 postinstall .../bitsy-boilerplate
> npm run fetch-bitsy


> bitsy-boilerplate@1.0.0 fetch-bitsy .../bitsy-boilerplate
> node --experimental-modules fetch-bitsy-source.mjs

internal/modules/run_main.js:54
    internalBinding('errors').triggerUncaughtException(
                              ^

TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".json" for .../bitsy-boilerplate/bitsy-paths.json
    at Loader.defaultGetFormat [as _getFormat] (internal/modules/esm/get_format.js:65:15)
    at Loader.getFormat (internal/modules/esm/loader.js:113:42)
    at Loader.getModuleJob (internal/modules/esm/loader.js:244:31)
    at async ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:44:17)
    at async Promise.all (index 4)
    at async link (internal/modules/esm/module_job.js:48:9) {
  code: 'ERR_UNKNOWN_FILE_EXTENSION'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bitsy-boilerplate@1.0.0 fetch-bitsy: `node --experimental-modules fetch-bitsy-source.mjs`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bitsy-boilerplate@1.0.0 fetch-bitsy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     .../.npm/_logs/2020-05-09T19_07_10_438Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bitsy-boilerplate@1.0.0 postinstall: `npm run fetch-bitsy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bitsy-boilerplate@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     .../.npm/_logs/2020-05-09T19_07_10_595Z-debug.log
```

I'm on OS X 10.12, running node 14.2.0 and npm 6.14.4

Any ideas what might be going wrong? Thank you :)

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "watch-html": "npm-watch build-html",
     "build": "npm run build-hacks&&npm run build-html",
     "start": "concurrently 'npm:watch-*'",
-    "fetch-bitsy": "node --experimental-modules fetch-bitsy-source.mjs",
+    "fetch-bitsy": "node --experimental-modules --experimental-json-modules fetch-bitsy-source.mjs",
     "postinstall": "npm run fetch-bitsy"
   },
   "author": "Sean S. LeBlanc & Elkie Nova",

